core/genesis.go

```diff
@@ -380,7 +380,7 @@ func DefaultGardenGenesisBlock() *Genesis {
 		Nonce:      66,
```

Comment:
I notice there is no `DefaultColosseumGenesisBlock()`. I think `DefaultGenesisBlock()` as it exists today should be renamed to `DefaultColosseumGenesisBlock()`, and then `DefaultGenesisBlock()` can just wrap that function until a later mainnet function is added... IMO

Reply:
Changed it.